README.md

```diff
@@ -71,6 +72,48 @@ See an alternative video showing my cluster running with KinD on my Mac and the
 
 [Try the step-by-step tutorial](https://blog.alexellis.io/ingress-for-your-local-kubernetes-cluster/)
 
+## Running in-cluster, using Google Compute Engine for the exit node using helm
+
+> Note: this example is now multi-arch, so it's valid for `x86_64`, `ARMHF`, and `ARM64`.
+
+If you do not have helm installed and configured follow the instructions [here](https://github.com/openfaas/faas-netes/blob/master/HELM.md)
+
+It is assumed that you have gcloud installed and configured on your machine. If not follow the instructions [here](https://cloud.google.com/sdk/)
+
+```sh
+# Get current projectID
+export PROJECTID=$(gcloud config get-value core/project 2>/dev/null)
+
+# Create a service account
+gcloud iam service-accounts create inlets --display-name "inlets"
+
+# Get inlets service account email
+export SERVICEACCOUNT=$(gcloud iam service-accounts list | grep inlets | awk '{print $2}')
+
+# Assign appropriate roles to inlets service account
+gcloud projects add-iam-policy-binding $PROJECTID \
+--member serviceAccount:$SERVICEACCOUNT \
+--role roles/compute.admin
+
+gcloud projects add-iam-policy-binding $PROJECTID \
+--member serviceAccount:$SERVICEACCOUNT \
+--role roles/iam.serviceAccountUser
+
+# Create inlets service account key
+gcloud iam service-accounts keys create ./key.json \
+--iam-account $SERVICEACCOUNT
+
+# Create a secret to store the service account key file
+kubectl create secret generic inlets-access-key --from-file=inlets-access-key=key.json
+
+# Clone inlets-operator repo
+git clone https://github.com/inlets/inlets-operator.git
```

Comment:
We have a helm chart repo now, so we can modify this example and the one I added too.